python/graphstorm/model/multitask_gnn.py

```diff
@@ -553,14 +553,14 @@ def multi_task_mini_batch_predict(
                                                     device,
                                                     return_proba,
                                                     return_label)
-                    assert labels is None or len(labels) == 1, \
+                    assert not labels or len(labels) == 1, \
```

Comment:
Any reason we make the change here? Ask because ``not labels`` control larger scope than ``labels is None``.

Reply:
not labels can handle the empty dict cases. If the labels is `{}`, it will not crash the program.